build/dangerjs/detectNonApprovedDependencies/utils/getRuntimeDependencies.ts

```diff
@@ -0,0 +1,30 @@
+import { spawnSync } from 'child_process'
+
+import config from '../../../../config'
+
+const { paths } = config
+
+const getRuntimeDependencies = () => {
+  const dependencyRegex = /^dependency:\s+(.*)$/
+  const result = spawnSync("yarn gulp test:dependencies:list --prefix='dependency: '", {
```

Comment:
unfortunately, it is necessary to introduce this complication of running weback in a separate process - otherwise, if run in DangerJS's process, `license-webpack-plugin` won't detect dependencies (presumably, because of the way how DangerJS handles imports, but for the sake of saving time haven't made a deep dive into this)